packages/grafana-data/src/transformations/matchers/refIdMatcher.ts

```diff
@@ -14,7 +14,7 @@ const refIdMatcher: FrameMatcherInfo<string> = {
   get: (pattern: string) => {
     const regex = stringToJsRegex(pattern);
     return (frame: DataFrame) => {
-      return regex.test(frame.refId || '');
+      return regex.test(frame.refId || '') || pattern === frame.refId;
```

Comment:
would we want to check that frame.refId has a value first before checking it against pattern? or check that pattern is not an empty string? Not sure what's all plausible here as weird values we wouldnt want to count.

Reply:
Maybe I'm not fully understanding the suggestion here, but if frame.refId is undefined, it will always return false, which is okay.🤔